UIDokan/Sources/UIDokan/Extensions/UILabel+Extension.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  File.swift
+//  
+//
+//  Created by Omar Ahmed on 21/07/2022.
+//
+
+import UIKit
+
+public extension UILabel {
+    
+    func addTrailing(image: UIImage, text:String) {
+        let attachment = NSTextAttachment()
+        attachment.image = image
+        attachment.bounds = CGRect(x: 0, y: -5, width: (attachment.image?.size.width)!, height: (attachment.image?.size.height)!)
```

Comment:
Force unwrap is prohibited! You owe a Mega Ice Cream 🍦😁

I like the idea behind this TBH ❤️
- You may add size as an input to the method. 
- How y is -5.. It's not a frame and should be zero

Reply:
Please document this method this it's mandatory to tell the developer that you're updating the attributed string.